### PR TITLE
Optimise Delete Function

### PR DESCRIPTION
**Pull request by meatyturtle**
Summary of changes:
Optimise Confirm Delete dialog to match format and style of Help dialog
Optimise Confirm Delete dialog for arrow keys usage
Output consistent error messages for invalid Delete command format 

PS: Current 'consistent' message does not display an example of a valid format, kiv for future iterations
Code coverage is also poor because GUI-related code are unable to be covered by tests

good night

### CODE REVIEW

**Comment by fymbc** [comment]
LGTM. Noted that CodeCov is low due to adding GUI-related code.

File: src/main/resources/images/warning_icon.png
Comment (Shantaa) [inline src/main/resources/images/warning_icon.png:1]: Appropriately chosen.

File: src/main/resources/view/ConfirmDeleteWindow.css
Comment (Shantaa) [inline src/main/resources/view/ConfirmDeleteWindow.css:1]: Well done on including a css file to ensure good user experience!

**Review by Shantaa — COMMENTED** [review]
Looks good! Window comes out nicely.

**Pull request merged**